README.md

```diff
@@ -0,0 +1,24 @@
+# TraX
+
+### TraX is a utility to assist in finding files on the BeOS.
+TraX is not a "query" tool. It does not keep a database of files and attributes.
+It is a tradition find tool. It searches the disk every time, looking for files that meet your criteria.
+The find-criteria are not BeOS specific, and it does not (currently) search attributes.
+
```

Comment:
Add the info from the "Read This First" that there's "FastTraX" and "TraX" and their differences.